src/objects/conf/base.py

```diff
@@ -47,77 +17,14 @@
     }
 }
 
-DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
-
-CACHES = {
-    "default": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://{config('CACHE_DEFAULT', 'localhost:6379/0')}",
-        "OPTIONS": {
-            "CLIENT_CLASS": "django_redis.client.DefaultClient",
-            "IGNORE_EXCEPTIONS": True,
-        },
-    },
-    "axes": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://{config('CACHE_AXES', 'localhost:6379/0')}",
-        "OPTIONS": {
-            "CLIENT_CLASS": "django_redis.client.DefaultClient",
-            "IGNORE_EXCEPTIONS": True,
-        },
-    },
-    "oidc": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://{config('CACHE_OIDC', 'localhost:6379/0')}",
-        "OPTIONS": {
-            "CLIENT_CLASS": "django_redis.client.DefaultClient",
-            "IGNORE_EXCEPTIONS": True,
-        },
-    },
-}
 
 # Application definition
 
-INSTALLED_APPS = [
-    # Note: contenttypes should be first, see Django ticket #10827
-    "django.contrib.contenttypes",
-    "django.contrib.auth",
-    "django.contrib.sessions",
-    # Note: If enabled, at least one Site object is required
-    "django.contrib.sites",
-    "django.contrib.messages",
-    "django.contrib.staticfiles",
-    # django-admin-index
-    "ordered_model",
```

Comment:
That's right, `ordered_model` is actually unused in the projects that use `open-api-framework` for settings thus far (and I think OZ and ON don't actually use it either), it was in there from default project

Reply:
Seems like it might actually be used by `django-admin-index:` https://github.com/maykinmedia/django-admin-index?tab=readme-ov-file#usage

It crashes for me if I manually try to edit app groups in `/admin/admin_index/appgroup/`

Reply:
I upgraded OAF and this is now fixed